data_import.py

```diff
@@ -0,0 +1,35 @@
+# python script to import csv data into our database
+import csv
+import os
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.sql import text
+from models.PreviousSession import PreviousSession
+from models.UpcomingSession import UpcomingSession
+from models.SprintSession import SprintSession
+
+db_engine = create_engine(os.getenv("DATABASE_URL"))
+db_session = scoped_session(
+    sessionmaker(autocommit=False, autoflush=False, bind=db_engine)
+)
+
+def import_data(file_name, model_class):
+    with open(file_name, "r") as file:
+        rows = csv.DictReader(file)
+        for row in rows:
+            # Create an instance of the model class
+            session_obj = model_class(**row)  # Unpack row data using double splat operator
+            db_session.add(session_obj)
+        db_session.commit()
+
+def main():
+    db_session.execute(text("TRUNCATE TABLE previous_sessions"))
+    db_session.execute(text("TRUNCATE TABLE upcoming_sessions"))
+    db_session.execute(text("TRUNCATE TABLE sprint_sessions"))
```

Comment:
This is pretty dangerous if accidentally run. Should this have an interactive "are you sure to truncate all tables" confirmation?